src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/TestUtils.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.DocumentTag;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.DocumentWithMetadata;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.Document;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.IdValue;
+
+public class TestUtils {
+
+    private TestUtils(){
+
+    }
+
+    public static List<IdValue<DocumentWithMetadata>> getDocumentWithMetadata(String docId, String filename,
+                                                                               String description, DocumentTag tag) {
+        String documentUrl = "http://dm-store/" + docId;
```

Comment:
Can we have the dm-store url as a private variable of this class rather than hardcoded here?

Reply:
Since this is only used in this function, I think it is better as a local variable.